src/rules/converters/space-within-parens.ts

```diff
@@ -0,0 +1,14 @@
+import { RuleConverter } from "../converter";
+
+export const convertSpaceWithinParens: RuleConverter = tslintRule => {
+    return {
+        rules: [
+            {
+                ...(tslintRule.ruleArguments.length !== 0 && {
+                    ruleArguments: tslintRule.ruleArguments,
```

Comment:
https://eslint.org/docs/rules/space-in-parens doesn't show the ESLint `space-in-parens` rule as being able to receive a numeric literal the same way as the TSLint rule. It instead has `always` or `never` as options. This'll need to convert from the TSLint way to the ESLint way.

Reply:
Ah, I was looking at the tslint docs and didn't look at eslint. I will update this to convert appropriately.